### DISCUSSION TITLE
[Projects beta] API to find project item through pull request / issue

### DISCUSSION THREAD

**Discussion by gr2m** [community/community#6320] [Projects and Issues]
Given a pull request, I want to find if it is assigned to a Memex project, and if so, I want to find all the fields set on the project item.

I'd like to query something like this

```graphql
query ($pullRequestId: ID!) {
  node(id: $pullRequestId) {
    ... on PullRequest {
      projectNextItems {
        project {
          title
        }
        fieldValues(first: 5) {
          nodes {
            projectField {
              name
              settings
            }
            value
          }
        }
      }
    }
  }
}
```

But as far as I can tell from https://docs.github.com/en/issues/trying-out-the-new-projects-experience/using-the-api-to-manage-projects and the GraphQL schema when setting the `GraphQL-Features: projects_next_graphql` header no such connection exists today

**Comment by widal001** [comment]
This would also be very helpful for accessing project values on an issue as well. Currently you can go from an issue to a `ProjectV2` that it's been assigned to using the GraphQL, but you can't find the values of the associated `ProjectV2Item` without querying all items in that project.
